ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -245,15 +245,6 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
-    {
-      id: 'P3N Searing Breeze',
-      type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '66B6', source: 'Phoinix', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ id: '66B6', source: 'Phoinix', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ id: '66B6', source: 'Protophénix', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ id: '66B6', source: 'フェネクス', capture: false }),
-      response: Responses.stackThenSpread(),
```

Comment:
P3N Searing Breeze is the same as P2N Sewage Eruption, a series of ground circles.  There's no stack here or spread.

Reply:
It occurs to me that I added a similar situation in E6n. I'll maybe go change that later tonight.